templates/category/index.html

```diff
@@ -12,6 +12,25 @@
 
 {% block title %}{{ category }} — {{ block.super }}{% endblock %}
 
+{% block meta_tags %}
+    <meta property="og:title" content="{{ category }}">
+    <meta property="og:description" content="{{ category.description }}">
+    <meta property="og:image" content="{% static "images/block1.png" %}">
+
+    {% with category_url=category.get_full_absolute_url %}
```

Comment:
You have access to `request` here which can convert relative URLs to absolute ones, you don't need `category.get_full_absolute_url`.